src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,7 @@
-export const UserInfo = () => {};
+export const UserInfo = ({ user }) => {
```

Comment:
The `UserInfo` component is missing type definitions for its `user` prop. Consider defining a TypeScript interface or type for the `user` object to ensure type safety and clarity.